organization/index.d.ts

```diff
@@ -0,0 +1,30 @@
+import { Address } from "..";
+import { EventTicket } from "../event/ticket";
+import { User } from "../user";
+
+// Organization
+export type Organization = {
+  id: string;
+  name: string;
+  socials: OrganizationSocial[];
+  members: OrganizationMember[];
+  address?: Address;
+  savedTickets: EventTicket[];
+  events: Event[];
+  updatedAt: Date;
+  createdAt: Date;
+};
+
+export type OrganizationSocial = {
+  type: OrganizationSocialType;
+  url: string;
+};
+
+export type OrganizationSocialType = "facebook" | "twitter" | "instagram" | "linkedin" | "youtube" | "website" | string;
```

Comment:
Wouldn't it be better in Enum form?

Reply:
In the database and graphql for optimization, theses types will be oriented in this idea of conception.
Let's keep this use before we change it later